tests/test_main.py

```diff
@@ -16,6 +18,7 @@ def test_generate_creates_valid_yml():
     """Test that ensures that the generate command creates
     the .yml file correctly."""
     result = runner.invoke(main.app, ["generate"])
+    print(result.stdout)
```

Comment:
Are these debugging statements? If so, and they are needed, could we add comments to clarify why they are necessary?